README.md

```diff
@@ -373,18 +356,36 @@ All images are pushed to [Docker Hub](docker_hub_tags).
 
 Fixed version tags (`x.y.z`) are frozen: we never update them.
 
+Example of minor versions timeline:
+
+- `x.y.0` - Alpha
+-   | -> `x.y.1` - Beta
+-     | -> `x.y.2` - Stable
+-       | -> `x.y.3` - Update
```

Comment:
We have no 'update' maturity status in our [docs](https://www.tarantool.io/en/doc/latest/dev_guide/release_management/). I would mark it just as Stable, the same as .2.